### DISCUSSION TITLE
Editing Tools and fun menu

### DISCUSSION THREAD

**Discussion by Raj6544** [PrajjwalDatir/Kaoi#162] [Ideas]
What about adding of some editing tools like text to image editing tools. Many kind of editing tools and logo maker. also Adding many kinda filters in videos and images.
In fun menu, can be added tic tac toe game,
Truth or dare game, checking beautiful ness, handsomeness etc.

**Comment by PrajjwalDatir** [comment]
> editing tools like text to image editing tools. Many kind of editing tools and logo maker. also Adding many kinda filters in videos and images.

Well, this makes the bot heavier, but we can do it.
https://pixelixe.com/docs/v2/
This can be used to do image related functions.

> In fun menu, can be added tic tac toe game,
Truth or dare game, checking beautifulness, handsomeness etc.

Yes, sure! If you are a developer, let me know if you need any help in developing this commands

**Reply by Raj6544** [reply]
Well,I'm not a developer. I just my shared ideas. If these would be added that's why. 😅😅

**Comment by ghost** [comment]
How About Adding A Guess The Character Feature From Images?
